srcpkgs/kubernetes-helm/template

```diff
@@ -70,7 +70,6 @@ do_install() {
 
 	# Copy binaries and license
 	vbin _dist/linux-${_go_target_arch}/helm
-	vbin _dist/linux-${_go_target_arch}/rudder
```

Comment:
The `rudder` binary is no longer is built. I'm not sure it was ever supposed to be packaged. The release notes state:
```
Both compiled binaries of helm and tiller are now distributed with each release
```
This indicates to me that only `helm` and `tiller` are meant to be packaged.
In addition, I find no mention of the `rudder` executable in the helm docs.